lib/client.js

```diff
@@ -1,6 +1,7 @@
 const { URLSearchParams } = require('url');
 const { authFromCapability } = require('./smart');
 const HttpClient = require('./http-client');
+const { Pager } = require('./pager');
```

Comment:
perhaps "Paginator" ?    (A "Pager" is a device popular in the 80's for calling back phone numbers sent to it)